.vscode/settings.json

```diff
@@ -1,4 +1,13 @@
 {
     "java.configuration.updateBuildConfiguration": "disabled",
-    "typescript.tsdk": "node_modules/typescript/lib"
+    "typescript.tsdk": "node_modules/typescript/lib",
+    "jestrunner.debugOptions": {
```

Comment:
jest parallelizes tests, but for that it uses JSON.stringify, which can't serialize bigint. it's needed to run-in-band, or with max-workers = 1.

Reply:
-> https://github.com/jestjs/jest/issues/11617